scripts/reference-tables.js

```diff
@@ -0,0 +1,406 @@
+/*
+*   This content is licensed according to the W3C Software License at
+*   https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document
+*
+*   File:   reference-tables.js
+*/
+
+var fs = require('fs');
+var i;
+
+var fileNameTemplate = 'reference-tables.template';
+var fileNameIndex = '../examples/index.html';
+var examplesDirectory = '../examples/';
+
+var ariaRoles = [
+  'application',
+  'article',
+  'banner',
+  'button',
+  'cell',
+  'checkbox',
+  'columnheader',
+  'combobox',
+  'complementary',
+  'contentinfo',
+  'definition',
+  'directory',
+  'document',
+  'feed',
+  'figure',
+  'form',
+  'grid',
+  'gridcell',
+  'group',
+  'heading',
+  'img',
+  'link',
+  'list',
+  'listbox',
+  'listitem',
+  'main',
+  'math',
+  'menu',
+  'menubar',
+  'menuitem',
+  'menuitemcheckbox',
+  'menuitemradio',
+  'navigation',
+  'none',
+  'note',
+  'option',
+  'presentation',
+  'progressbar',
+  'radio',
+  'radiogroup',
+  'row',
+  'rowgroup',
+  'rowheader',
+  'scrollbar',
+  'search',
+  'searchbox',
+  'separator',
+  'slider',
+  'spinbutton',
+  'switch',
+  'tab',
+  'table',
+  'tablist',
+  'tabpanel',
+  'term',
+  'textbox',
+  'toolbar',
+  'tooltip',
+  'tree',
+  'treegrid',
+  'treeitem'
+];
+
+var ariaPropertiesAndStates = [
+  'aria-activedescendant',
+  'aria-atomic',
+  'aria-atomic',
```

Comment:
Duplicate. There is a few more below, but I'll skip commenting them each individually